translations/ui-finance/en.json

```diff
@@ -263,7 +263,7 @@
   "ledger.unavailable": "Unavailable",
   "ledger.available": "Available",
   "ledger.name": "Name",
-  "ledger.code": "Abbreviation",
```

Comment:
@dltj could you update translations for it. thanks

Reply:
https://github.com/folio-org/ui-finance/pull/161 happening now.

Reply:
👍